tests/ui/asm/unsupported-option.rs

```diff
@@ -0,0 +1,10 @@
+//@ run-rustfix
```

Comment:
This probably needs `//@ needs-asm-support` otherwise it will fail on targets without asm support.